security/src/main/java/nl/nn/adapterframework/lifecycle/servlets/ServletAuthenticatorBase.java

```diff
@@ -92,7 +114,8 @@ private SecurityFilterChain createSecurityFilterChain() {
 		return configureHttpSecurity(httpSecurityConfigurer);
 	}
 
-	private SecurityFilterChain configureHttpSecurity(HttpSecurity http) {
+	@Override
+	public SecurityFilterChain configureHttpSecurity(HttpSecurity http) {
```

Comment:
I can't select line 122 to add a comment, but perhaps you can add a comment there stating something like:
`http.csrf().disable(); // Disabled because the front-end doesn't support CSFR tokens atm`